src/Microsoft.DotNet.Build.Tasks/Targets/localization.targets

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="MultilingualAppToolkit">
+    <MultilingualAppToolkitVersion>4.0</MultilingualAppToolkitVersion>
+    <MultilingualFallbackLanguage>en-US</MultilingualFallbackLanguage>
+  </PropertyGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.de.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.es.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.fr.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.it.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.ja.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.ko.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.ru.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.zh-Hans.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildProjectDirectory)\Resources\Strings.zh-Hant.resx">
+      <Visible>True</Visible>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
They do show up correctly in VS, but only with "Visible" explicitly set to True.

I agree with conditioning the importing of the localization target on the existence of an .xlf file.  I don't think we have a case where a project would only have some of the .xlf or localized .resx files; if we see later that we do actually have cases like that, we can make a change at that point.